deploy/kubernetes/setup-csi-snapshotter.yaml

```diff
@@ -72,33 +72,32 @@ spec:
       serviceAccount: csi-snapshotter
       containers:
         - name: csi-provisioner
-          image: quay.io/k8scsi/csi-provisioner:v1.3.0
+          image: csi-provisioner:testsplitcontroller #quay.io/k8scsi/csi-provisioner:testsnapshotbeta #v1.3.0
           args:
-            - "--provisioner=csi-hostpath"
+            - "--v=5" #- "--provisioner=csi-hostpath"
             - "--csi-address=$(ADDRESS)"
-            - "--connection-timeout=15s"
           env:
             - name: ADDRESS
               value: /csi/csi.sock
-          imagePullPolicy: Always
+          imagePullPolicy: IfNotPresent #Always
           volumeMounts:
             - name: socket-dir
               mountPath: /csi
-        - name: csi-snapshotter
-          image: quay.io/k8scsi/csi-snapshotter:v1.2.0
+        - name: sidecar-csi-snapshotter
+          image: sidecar-snapshotter:v1beta1 #quay.io/k8scsi/csi-snapshotter:v1.1.0
```

Comment:
I would still call the name and image "csi-snapshotter" to make it consistent with the other sidecars.

Reply:
sure